float32[18] lidar
int16 angle16
int8 angle8
int16 pitch
int16 roll
int16[6] mag
int16[6] acc
int16[6] gyro
int16 temp
